packages/mobile/src/components/challenge-rewards-drawer/ChallengeRewardsDrawer.tsx

```diff
@@ -337,7 +338,7 @@ export const ChallengeRewardsDrawer = ({
                 />
               ]
             : null}
-          {claimedAmount > 0 && challengeState !== 'disbursed' ? (
+          {claimedAmount > 0 && hasDisbursed ? (
```

Comment:
was this a bug then? cuz logic has reversed

Reply:
whoops nope this is error, good catch!